homeassistant/components/rki_covid/__init__.py

```diff
@@ -0,0 +1,172 @@
+"""RKI Covid numbers integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import aiohttp
+import async_timeout
+from rki_covid_parser.parser import RkiCovidParser
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, update_coordinator
+
+from .const import DOMAIN
+from .data import DistrictData, StateData
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["sensor"]
```

Comment:
Use Platform enum https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L22

Reply:
done

Reply:
It seems you reverted this change?

Reply:
He never committed it

Reply:
I did, but had to revert it due to an error while running locally.
[9f2d1a2cb4dcff7dac7d0c6bd699afd46ba1ebf6](https://github.com/home-assistant/core/pull/60506/commits/9f2d1a2cb4dcff7dac7d0c6bd699afd46ba1ebf6)

But now I changed it again now. Sorry for the confusion